jenkins/debian/Jenkinsfile_v4l2

```diff
@@ -0,0 +1,36 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
+
+buildTest {
+
```

Comment:
Pipelines should probably best described as [scripted pipelines](https://jenkins.io/doc/book/pipeline/#scripted-pipeline-fundamentals)